marlowe-runtime-cli/app/Language/Marlowe/Runtime/CLI/Command/Tx.hs

```diff
@@ -35,12 +37,21 @@ txCommandParser subCommandParser = TxCommand
       , metavar "FILE_PATH"
       , help "Sign the transaction manually. Writes the CBOR bytes of the unsigned transaction to the specified file for manual signing. Use the submit command to submit the signed transaction."
       ]
-    metadataFileParser = optional $ strOption $ mconcat
-      [ long "metadata-file"
-      , short 'm'
-      , help "A JSON file containing a map of integer indexes to arbitrary JSON values that will be added to the transaction's metadata."
-      , metavar "FILE_PATH"
-      ]
+    metadataFileParser
+      | metadataSupported = optional $ strOption $ mconcat
+        [ long "metadata-file"
+        , short 'm'
+        , help "A JSON file containing a map of integer indexes to arbitrary JSON values that will be added to the transaction's metadata."
+        , metavar "FILE_PATH"
+        ]
+      | otherwise = pure Nothing
+    tagsFileParser
+      | metadataSupported = optional $ strOption $ mconcat
+        [ long "tags-file"
+        , help "A JSON file containing a map of tags indexes to optional JSON-encoded metadata values that will be added to the transaction's 1564 metadata key."
```

Comment:
We should probably note here that the tags will override conflicting parts of the normal metadata.